### ISSUE TITLE
djangocms-installer : requirements.txt can't fully be reused (a dependency is not available with pip).

### ISSUE THREAD

**Issue by vmonteco** [divio/django-cms#4571]
I've installed djangocms with the `djangocms-installer` pip package (using default choices).

Here is the output of `pip freeze` :

    dj-database-url==0.3.0
    Django==1.7.10
    django-appconf==1.0.1
    django-classy-tags==0.6.2
    django-cms==3.1.3
    django-reversion==1.8.5
    django-sekizai==0.8.2
    Django-Select2==5.2.0
    django-treebeard==3.0
    djangocms-admin-style==0.2.8
    djangocms-column==1.5
    djangocms-file==0.1
    djangocms-flash==0.2.0
    djangocms-googlemap==0.3
    djangocms-inherit==0.1
    djangocms-installer==0.8.0
    djangocms-link==1.7.0.dev1
    djangocms-picture==0.1
    djangocms-style==1.5
    djangocms-teaser==0.1
    djangocms-text-ckeditor==2.6.0
    djangocms-video==0.1
    html5lib==0.9999999
    Pillow==3.0.0
    pytz==2015.6
    six==1.10.0
    tzlocal==1.2
    wheel==0.24.0

`djangocms-link==1.7.0.dev1` can't be found by `pip` even if I try something like `pip install [--allow-external djangocms-link==1.7.0.dev1] [--allow-unverified djangocms-link==1.7.1.dev1] djangocms-link==1.7.0.dev1`.
It simply says :

    Could not find a version that satisfies the requirement djangocms-link==1.7.0.dev1 (from -r requirements.txt (line 17)) (from versions: 1.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4, 1.0.5, 1.0.6a1, 1.1, 1.1.1, 1.1.2, 1.1.3, 1.2, 1.3, 1.3.1, 1.3.2, 1.3.3, 1.3.4, 1.3.5, 1.4.0, 1.4.1, 1.5, 1.6, 1.6.1, 1.6.2)
No matching distribution found for djangocms-link==1.7.0.dev1 (from -r requirements.txt (line 17))

Why doesn't djangocms-installer use packages available with pip?

How could I fix this? Could an other version do the job?

(I tried and It looks like using the github repository address for djangocms-link can do the job wouldn't it be better to use directly a version available on pip like for the other packages?)

I tried with python3.5 (But django1.7 isn't supported with this one), and python3.4.
Pip version is 7.1.2

Thank you.

**Comment by yakky** [comment]
@vmonteco this actually belongs to https://github.com/nephila/djangocms-installer, could you reopen it there?
TL;DR: installer uses github version of plugins in some cases, use dump mode (http://djangocms-installer.readthedocs.org/en/latest/usage.html#dump-mode) in case you want to reuse requirements
